docs/database-setup.md

```diff
@@ -11,16 +11,19 @@ npm install --save-dev sequelize-cli
 
 ```bash
 npx sequelize-cli init
-```
-
 ## Generate Migrations
 
 ```bash
 npx sequelize-cli migration:generate --name create-tables
 ```
 
+Note: Sequelize CLI does not automatically generate migrations from models. You need to manually create migration files using the `sequelize-cli migration:generate` command.
```

Comment:
I understand that. But then I need to write the code in the migrations myself. Isn't there a better way where the migration is inferred if possible from the model structure. e.g. if a column is added a migration that just adds that column is made. Same for tables and other simple additive things. (you can just reply w/o updating this file)